Automattic/wp-calypso
# Marketing Connections: Add Mastodon

#### Proposed Changes

* This adds the option to connect to a Mastodon instance.

![image](https://user-images.githubusercontent.com/24195829/216421289-b0ee793c-2102-4718-9446-45667ad80acb.png)


#### Testing Instructions



* This should be tested alongside D93584-code (which also needs to be reviewed).
* Go to `/marketing/connections/yoursite.com`.
* Try to connect to a Mastodon instance.

Related discussion and more detailed testing instructions: pewaj1-3x-p2

#### Pre-merge Checklist
